addons/l10n_ar/demo/account_supplier_refund_demo.xml

```diff
@@ -23,4 +23,17 @@
 
     <function model="account.move.reversal" name="reverse_moves" eval="[ref('demo_sup_refund_invoice_4')]"/>
 
+    <!-- Liquido Producto document vendor bill refund -->
+    <record id="demo_sup_refund_invoice_5" model="account.move.reversal" context="{'allowed_company_ids': [ref('company_ri')]}">
+        <field name="reason">demo_sup_refund_invoice_5: liquido producto bill refund (credit note)</field>
+        <field name="refund_method">cancel</field>
+        <field name="move_ids" eval="[(4, ref('demo_sup_invoice_8'), 0)]"/>
+        <field name="l10n_latam_document_type_id" ref="l10n_ar.dc_liq_cd_sp_a"/>
+        <field name="l10n_latam_document_number">00011-00000012</field>
+        <field name="date" eval="time.strftime('%Y-%m')+'-01'"/>
+        <field name="journal_id" model="account.move.reversal" eval="obj().env.ref('l10n_ar.demo_sup_invoice_8').journal_id"/>
```

Comment:
@zaoral I am actually a bit confused how the model could be account.move.reversal

Reply:
Hi @jco-odoo 

In this case, we add the model because is required for the eval. We use the account.move.reversal model because it was the same as the record we are defining but could be any model.

We actually don't care about the model, because we are using it only define the tag and call obj().env.ref() in the eval that does not rely on model. Maybe I am wrong about this please let me know. And If you like I can change it to account.journal?

Wait for your feedback.

Best Regards
```suggestion
        <field name="journal_id" model="account.journal" eval="obj().env.ref('l10n_ar.demo_sup_invoice_8').journal_id"/>
```

Reply:
If we cannot just remove it...